lib/Controller/AssistantController.php

```diff
@@ -47,45 +49,105 @@ public function getTextProcessingTaskResultPage(int $taskId): TemplateResponse {
 			$response->throttle(['userId' => $this->userId, 'taskId' => $taskId]);
 			return $response;
 		}
-		$this->initialStateService->provideInitialState('task', $task->jsonSerialize());
+		$this->initialStateService->provideInitialState('task', $task->jsonSerializeCc());
 		return new TemplateResponse(Application::APP_ID, 'taskResultPage');
 	}
 
 	/**
-	 * @param string $input
+	 * @param int $taskId
+	 * @return DataResponse
+	 */
+	#[NoAdminRequired]
+	#[NoCSRFRequired]
```

Comment:
Any particular reason to disable CSRF here?

Reply:
Yeah, shouldn't be disabled here!